.circleci/config.yml

```diff
@@ -1,6 +1,6 @@
 references:
   envoy-build-image: &envoy-build-image
-    envoyproxy/envoy-build:d0cefa7f071dbd4ef24399c2db8656c3a5d8c3ef
+    envoyproxy/envoy-build:8246167b9d238797cbc6c03dccc9e3921c37617d
```

Comment:
@lizan do you think it would be easy/hard for us to move from CircleCI to Azure as well?

Reply:
shouldn’t be hard especially if the project is not that large so you can do without RBE